ostelco-ios-client/ViewControllers/Home/HomeViewController2.swift

```diff
@@ -10,13 +10,14 @@ import UIKit
 import Stripe
 import Siesta
 import ostelco_core
+import OstelcoStyles
+
+class HomeViewController2: ApplePayViewController {
```

Comment:
while you're mucking around in here you wanna drop the `2` off this? 😄

Reply:
done